solr/core/src/test/org/apache/solr/rest/schema/TestBulkSchemaAPI.java

```diff
@@ -125,7 +126,7 @@ public void testAnalyzerClass() throws Exception {
         "    'name' : 'myNewTextFieldWithAnalyzerClass',\n" +
         "    'class':'solr.TextField',\n" +
         "    'analyzer' : {\n" +
-        "        'luceneMatchVersion':'5.0.0',\n" +
+        "        '" + IndexSchema.LUCENE_MATCH_VERSION_PARAM + "':'5.0.0',\n" +
```

Comment:
Using Constants in tests like here is a bit too far IMO.  It slightly obscures readability and there's a point to be made that changing the input/output constant _should_ break a test.  Subjective, I know.   I don't mean to suggest the removal of all constants in tests but here in particular it's ugly.